### ISSUE TITLE
Should we have an issue that informs us a new OpenSSL or wxWidgets version is out?

### ISSUE THREAD

**Issue by paulo-ferraz-oliveira** [erlef/otp_builds#5]
So we could sync. CI and be "on top of latest"...

**Comment by paulo-ferraz-oliveira** [comment]
Or, more generally, how should we implement in such a way that we can rebuild older artifacts with newer OpenSSL versions?

**Comment by wojtekmach** [comment]
OTP team doesn't rebuild artifacts and so I wouldn't either. I don't think we care if there's a new OpenSSL or wxWidgets version, we don't need to be absolutely up to date in my opinion. What I'd do instead is see what openssl/wx version OTP builds for Windows are using so we're consistent.

**Comment by paulo-ferraz-oliveira** [comment]
> OTP team doesn't rebuild artifacts and so I wouldn't either

But they don't statically link OpenSSL for you either, unless you want, right? But I agree we can live without it and use the same versions as they do for Windows.

I'll leave this open, to think of a mechanism to obtain those versions and warn us. Sounds Ok?

**Comment by wojtekmach** [comment]
OTP builds for windows are statically linking openssl and wxwidgets.

**Comment by paulo-ferraz-oliveira** [comment]
Ah, I wasn't thinking about Windows specifically, but you're right. If you feel we can accompany those without some sort of automated mechanism, feel free to close. I just wanted to understand if it's something we wanted.

**Comment by wojtekmach** [comment]
Yeah, I'd like to keep this open and add some automatic check that ensures we use the same versions as OTP. Can be a nightly job or whatever. It's a low priority but would be nice to have imo.
